_includes/releases/v23.1/v23.1.0-rc.1.md

```diff
@@ -0,0 +1,78 @@
+- The metric description of `rebalancing.readbytespersecond` and `rebalancing.writebytespersecond` now correctly reference bytes read and bytes written respectively. [#101710][#101710]
+- Fixed a rare bug introduced prior to v22.1 where distributed plans shutdown could cause the graceful drain of a node to become stuck retrying forever.This bug leads to errors like `drain details: distSQL execution flows:` with a non-zero number of flows that is not reducing over a long period of time. [#101884][#101884]
+- Fixed a bug where a `RESTORE` operation with `skip_localities_check` could fail with errors if regions were missing on a cluster. [#101798][#101798]
+- Fixed a bug introduced in testing releases of v23.1 that could cause incorrect results for queries with `STRICT` user-defined functions. [#101951][#101951]
+- Fixed a rare bug that can cause keys to be unexpectedly deleted locally within a store by replica rebalancing during a write heavy workload. [#102166][#102166]
+- Fixed a bug where a failed or cancelled `IMPORT` operation could leave some of the imported rows behind after it was cancelled, if the writing processes were slow enough to continue writing after the cleanup process started. [#102246][#102246]
+- Fixed a bug in the behavior of the `enforce_home_region` [session variable](../v23.1/set-vars) that may have allowed a hash join to be favored over a lookup join, or failed to error out remote accesses done by uniqueness checks for mutations on R`EGIONAL BY ROW` tables. Also, static erroring of some locality-optimized lookup joins is now handled dynamically during query execution. [#102287][#102287]
+- Fixed a bug introduced in testing releases of v23.1 where a node could crash when evaluating a `COPY` command when the schema had `INT2` or `INT4` type. [#102307][#102307]
+- Fixes a bug where a backup of a key's revision history may not correctly be restored to the proper revision of the key if it is split across multiple sorted string tables. [#102342][#102342]
+- Fixed a bug introduced in testing releases of v23.1 where a user could be prevented from logging in or viewing or changing `GRANT`s if the cluster had a long period of inactivity. [#102488][#102488]
+- Fixed a bug where a node that transfers a lease away and misses a liveness heartbeat could then erroneously update the closed timestamp during the stasis period if its liveness. This could lead to closed timestamp invariant violation and could cause the node to crash. In extreme cases, this could lead to inconsistencies in read-only queries. [#102602][#102602]
```

Comment:
```suggestion
- Fixed a bug where a node that transferred a lease away and missed a liveness heartbeat could then erroneously update the closed timestamp during the stasis period of its liveness. This could lead to closed timestamp invariant violation and could cause the node to crash. In extreme cases, this could lead to inconsistencies in read-only queries. [#102602][#102602]
```